antarest/tools/lib.py

```diff
@@ -233,12 +241,41 @@ def extract_commands(study_path: Path, commands_output_dir: Path) -> None:
         )
 
 
-def generate_diff(base: Path, variant: Path, output_dir: Path) -> None:
+def generate_diff(
+    base: Path, variant: Path, output_dir: Path, version: Optional[str] = None
+) -> None:
     if not output_dir.exists():
         output_dir.mkdir(parents=True)
     matrices_dir = output_dir / MATRIX_STORE_DIR
     matrices_dir.mkdir(exist_ok=True)
 
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        study_id = "temp_empty_study"
+        path_study = Path(tmp_dir) / study_id
+        matrix_path = Path(tmp_dir) / "matrix"
```

Comment:
not needed (see previous comment concerning the existing SimpleMatrixService instanciated after)